db/migrate/20210706181815_add_rekeyed_field_to_encrypted_fields.rb

```diff
@@ -0,0 +1,16 @@
+class AddRekeyedFieldToEncryptedFields < ActiveRecord::Migration[6.1]
```

Comment:
My concern is that this migration will probably freeze our database and it's adding another unnecessary column to each model, and we could refactor this to not need to add any columns at all. My suggestion is that you could check against the `updated_at` timestamp

Reply:
I didn't think that would provide enough resolution, i.e. what if something else updated the record recently? Then we wouldn't rekey it which would lead to unreadable data. 

The performance issue is gone in PG 11 https://dba.stackexchange.com/a/211222

Reply:
This is removed in favor of retrying from the beginning and skipping if already rekeyed.